core/src/main/java/org/apache/calcite/runtime/SqlFunctions.java

```diff
@@ -2214,7 +2250,7 @@ public static Collection multisetUnionAll(Collection collection1,
         inputTypes);
   }
 
-  private static Enumerable<FlatLists.ComparableList<Comparable>> p2(
+  private static Enumerable<ComparableList<Comparable>> p2(
```

Comment:
I do not think such kind of code cleaning is required here. Moreover not all places in the class are cleaned as well as in the rest of the code